test/testdata/cfg/keyword_args.rb.cfg.exp

```diff
@@ -6,7 +6,7 @@ subgraph "cluster_::<Class:<root>>#<static-init>" {
     "bb::<Class:<root>>#<static-init>_1" [shape = parallelogram];
 
     "bb::<Class:<root>>#<static-init>_0" [
-        label = "block[id=0, rubyBlockId=0]()\l<self>: T.class_of(<root>) = cast(<self>: NilClass, AppliedType {\l  klass = <S <C <U <root>>> $1>\l  targs = [\l    <C <U <AttachedClass>>> = SelfTypeParam(<S <C <U <root>>> $1><C <U <AttachedClass>>>)\l  ]\l});\l<hashTemp>$5: Symbol(:\"bar\") = :\"bar\"\l<hashTemp>$6: Symbol(:\"qux\") = :\"qux\"\l<magic>$7: T.class_of(<Magic>) = alias <C <Magic>>\l<statTemp>$4: {bar: Symbol(:\"qux\")} = <magic>$7: T.class_of(<Magic>).<build-keyword-args>(<hashTemp>$5: Symbol(:\"bar\"), <hashTemp>$6: Symbol(:\"qux\"))\l<returnMethodTemp>$2: T.untyped = <self>: T.class_of(<root>).foo(<statTemp>$4: {bar: Symbol(:\"qux\")})\l<finalReturn>: T.noreturn = return <returnMethodTemp>$2: T.untyped\l<unconditional>\l"
+        label = "block[id=0, rubyBlockId=0]()\l<self>: T.class_of(<root>) = cast(<self>: NilClass, AppliedType {\l  klass = <S <C <U <root>>> $1>\l  targs = [\l    <C <U <AttachedClass>>> = SelfTypeParam(<S <C <U <root>>> $1><C <U <AttachedClass>>>)\l  ]\l});\l<hashTemp>$5: Symbol(:\"bar\") = :\"bar\"\l<hashTemp>$6: Symbol(:\"qux\") = :\"qux\"\l<magic>$7: T.class_of(<Magic>) = alias <C <Magic>>\l<keywordArgsTemp>$4: {bar: Symbol(:\"qux\")} = <magic>$7: T.class_of(<Magic>).<build-keyword-args>(<hashTemp>$5: Symbol(:\"bar\"), <hashTemp>$6: Symbol(:\"qux\"))\l<returnMethodTemp>$2: T.untyped = <self>: T.class_of(<root>).foo(<keywordArgsTemp>$4: {bar: Symbol(:\"qux\")})\l<finalReturn>: T.noreturn = return <returnMethodTemp>$2: T.untyped\l<unconditional>\l"
```

Comment:
<img width="847" alt="Screen Shot 2020-08-07 at 2 50 32 PM" src="https://user-images.githubusercontent.com/5544532/89691269-5a158600-d8bd-11ea-80b8-fe880ca82d14.png">

The GitHub diff isn't very good because the line is long.